cmd/nerdctl/container/container_stop.go

```diff
@@ -41,6 +41,7 @@ func NewStopCommand() *cobra.Command {
 		SilenceErrors:     true,
 	}
 	stopCommand.Flags().IntP("time", "t", 10, "Seconds to wait before sending a SIGKILL")
+	stopCommand.Flags().StringP("signal", "s", "SIGTERM", "Signal to send to the container")
```

Comment:
The docs needs to be updated too https://github.com/containerd/nerdctl/blob/main/docs/command-reference.md#whale-nerdctl-stop